src/stores/printing.js

```diff
@@ -90,5 +100,20 @@ export const usePrintingStore = defineStore('printing', {
       }
       return { success, data, errors }
     },
+
+    /**
+     * fetches the list of workflows from traction
+     * @returns {Object}  {success: Boolean, data: Object, errors: Array} Is the fetch successful? The data and any errors
+     */
+    async fetchWorkflows() {
```

Comment:
Is  printing store a good place to keep it?Maybe the method can be placed in helper file as well for e.g lib/LabelPrintinHelpers.js with workflows stored inside the component as it is only used in one place.